pkg/server/apiextensions.go

```diff
@@ -67,16 +68,28 @@ func (c *inheritanceCRDLister) ListWithContext(ctx context.Context, selector lab
 	// Check for API inheritance
 	inheriting := false
 	inheritFrom := ""
-	// TODO(ncdc): for now, we are hard-coding "admin" as the logical cluster where all Workspace
-	// resources must live, at least for API inheritance to work.
-	const adminClusterName = "admin"
+	org, ws, err := helper.ParseLogicalClusterName(cluster.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	/* org could possibly be:
+		- admin (helper.OrganizationCluster)
+		- someorg
+
+	   If it's someorg, when looking for Workspace instances, we need to look in the logical cluster
```

Comment:
This will inform what we require as a virtual workspace - can we somehow hide it ... ?

Reply:
Virtual workspace, or just workspace in general?

Reply:
I was thinking virtual, since the question we want to answer is "show me the workspaces I need to know about"

Reply:
Client library with filtering would be fine too I guess